package.json

```diff
@@ -2,6 +2,7 @@
   "name": "protocol",
   "version": "0.1.0",
   "description": "UMA Protocol Smart Contracts",
```

Comment:
I know not related to this PR but is this description still accurate? should be `UMA protocol Mono repo` perhaps?

Reply:
Yeah, I'll send out a related PR to do some general updates on package.json files.